docs/_guides/codewind-vs-code-quick-guide.md

```diff
@@ -10,16 +10,16 @@ keywords: Codewind, VS Code, microservice
 ---
 
 ## Objectives
-* Install Visual Studio Code (VS Code) and Codewind
-* Develop a simple microservice that uses Eclipse Codewind on VS Code
+* Install Visual Studio Code (VS Code) and Codewind.
+* Develop a simple microservice that uses Eclipse Codewind on VS Code.
 
 ## Overview
-Use Eclipse Codewind to create application projects from `Application Stacks` that your company builds. With Codewind, you can focus on your code and not on infrastructure and Kubernetes. Application deployments to Kubernetes occur through pipelines when developers commit their local code to the correct Git repos Kabanero is managing via webhooks.
+Use Eclipse Codewind to create application projects from Application Stacks that your company builds. With Codewind, you can focus on your code and not on infrastructure and Kubernetes. Application deployments to Kubernetes occur through pipelines when developers commit their local code to the correct Git repos Kabanero is managing via webhooks.
```

Comment:
I know you mentioned that "via" had been corrected to "through," but it's still showing up in the latest commit. Could you please try changing it again? Thanks!